portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentsadministration/cpdocumentsandmedia/CPDocumentsandmedia.testcase

```diff
@@ -55,47 +55,10 @@
 		</execute>
 	</command>
 
-	<command name="AddDocument" priority="4">
```

Comment:
@anthony-chu Oh yeah, sorry I didn't make a note about that.  I checked with ee-7.0.x, and these were removed ([LRQA-31429](https://issues.liferay.com/browse/LRQA-31429)) there because of duplicate acceptance coverage on `EditDocument` and `EditFolder`, so I aligned the tests with the analysis from ([LRQA-30723](https://issues.liferay.com/browse/LRQA-30723)).  I think at some point they were accidentally re-added?